libraries/adaptive-expressions/src/expressionProperties/arrayExpression.ts

```diff
@@ -23,4 +23,20 @@ export class ArrayExpression<T> extends ExpressionProperty<T[]> {
     public constructor(value?: T[] | string | Expression) {
         super(value);
     }
+
+        /**
+     * Set an array value.
+     * @param value Value to set.
+     */
+    public setValue(value: T[] | string | Expression): void {
+        if (value !== undefined 
+            && value !== null 
```

Comment:
`!== undefined && !== null` can safely be collapsed to just `!= null` due to the way coercion works.

Reply:
Knowledge increased